### PR TITLE
[eclipse-temurin: update jdk11.0.16 to jdk11.0.17]

### PR DESCRIPTION
**Pull request by zdtsw**
update from jdk11.0.16 to jdk11.0.17

### CODE REVIEW

**Comment by gdams** [comment]
@zdtsw can you remove the added UBI dockerfiles, they're not ready for production yet

File: library/eclipse-temurin [hunk 1]
```diff
@@ -122,116 +122,129 @@ Constraints: nanoserver-1809, windowsservercore-1809
...
-Tags: 11.0.16.1_1-jdk-windowsservercore-ltsc2022, 11-jdk-windowsservercore-ltsc2022, 11-windowsservercore-ltsc2022
-SharedTags: 11.0.16.1_1-jdk-windowsservercore, 11-jdk-windowsservercore, 11-windowsservercore, 11.0.16.1_1-jdk, 11-jdk, 11
+Tags: 11.0.17_8-jdk-ubi9-minimal, 11-jdk-ubi9-minimal, 11-ubi9-minimal
+Architectures: amd64, arm64v8, ppc64le, s390x
+GitCommit: fa65b46d19f4db0e1c8736c3d7b2a8392dd58b1b
+Directory: 11/jdk/ubi/ubi9-minimal
+File: Dockerfile.releases.full
+
+Tags: 11.0.17_8-jdk-windowsservercore-ltsc2022, 11-jdk-windowsservercore-ltsc2022, 11-windowsservercore-ltsc2022
+SharedTags: 11.0.17_8-jdk-windowsservercore, 11-jdk-windowsservercore, 11-windowsservercore, 11.0.17_8-jdk, 11-jdk, 11
```
Comment (zdtsw) [inline library/eclipse-temurin:158]:
```suggestion
```

File: library/eclipse-temurin [hunk 2]
```diff
@@ -122,116 +122,129 @@ Constraints: nanoserver-1809, windowsservercore-1809
...
 File: Dockerfile.releases.full
 
-Tags: 11.0.16.1_1-jre-windowsservercore-ltsc2022, 11-jre-windowsservercore-ltsc2022
-SharedTags: 11.0.16.1_1-jre-windowsservercore, 11-jre-windowsservercore, 11.0.16.1_1-jre, 11-jre
+Tags: 11.0.17_8-jre-ubi9-minimal, 11-jre-ubi9-minimal
+Architectures: amd64, arm64v8, ppc64le, s390x
+GitCommit: fa65b46d19f4db0e1c8736c3d7b2a8392dd58b1b
+Directory: 11/jre/ubi/ubi9-minimal
+File: Dockerfile.releases.full
+
```
Comment (zdtsw) [inline library/eclipse-temurin:219]:
```suggestion
```

File: library/eclipse-temurin [hunk 3]
```diff
@@ -122,116 +122,115 @@ Constraints: nanoserver-1809, windowsservercore-1809
...
+Tags: 11.0.17_8-jdk-centos7, 11-jdk-centos7, 11-centos7
 Architectures: amd64, arm64v8, ppc64le
-GitCommit: 08dd7d416cee0fe00f1231351b521d577450133f
+GitCommit: fa65b46d19f4db0e1c8736c3d7b2a8392dd58b1b
 Directory: 11/jdk/centos
 File: Dockerfile.releases.full
 
-Tags: 11.0.16.1_1-jdk-windowsservercore-ltsc2022, 11-jdk-windowsservercore-ltsc2022, 11-windowsservercore-ltsc2022
-SharedTags: 11.0.16.1_1-jdk-windowsservercore, 11-jdk-windowsservercore, 11-windowsservercore, 11.0.16.1_1-jdk, 11-jdk, 11
 Architectures: windows-amd64
```
Comment (zdtsw) [inline library/eclipse-temurin:151]:
```suggestion
Tags: 11.0.17_8-jdk-windowsservercore-ltsc2022, 11-jdk-windowsservercore-ltsc2022, 11-windowsservercore-ltsc2022
SharedTags: 11.0.17_8-jdk-windowsservercore, 11-jdk-windowsservercore, 11-windowsservercore, 11.0.17_8-jdk, 11-jdk, 11
Architectures: windows-amd64
```

**Review by gdams — COMMENTED** [review]
LGTM

**Comment by tianon** [comment]
In the future, I'd recommend one PR at a time since the files are relatively short and often end up with merge conflicts otherwise. :see_no_evil:

**Pull request merged**